src/grpc/retry-interceptor.ts

```diff
@@ -77,45 +72,48 @@ export class RetryInterceptor {
 
     return (options, nextCall) => {
       let savedMetadata: Metadata;
-      let savedSendMessage: any;
-      let savedReceiveMessage: any;
-      let savedMessageNext: (arg0: any) => void;
+      let savedSendMessage: unknown;
+      let savedReceiveMessage: unknown;
+      let savedMessageNext: (arg0: unknown) => void;
       return new InterceptingCall(nextCall(options), {
         start: function (metadata, listener, next) {
           savedMetadata = metadata;
           const newListener: Listener = {
             onReceiveMessage: function (
-              message: any,
-              next: (arg0: any) => void
+              message: unknown,
+              next: (arg0: unknown) => void
             ) {
-              // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
               savedReceiveMessage = message;
               savedMessageNext = next;
             },
             onReceiveStatus: function (
               status: StatusObject,
+              // NOTE: we have to use `any` here because that is what is used in the grpc-js type definitions
+              // eslint-disable-next-line @typescript-eslint/no-explicit-any
               next: (arg0: any) => void
             ) {
               let retries = 0;
-              const retry = function (message: any, metadata: Metadata) {
+              const retry = function (message: unknown, metadata: Metadata) {
                 retries++;
                 const newCall = nextCall(options);
-                // eslint-disable-next-line @typescript-eslint/no-unsafe-argument
                 newCall.start(metadata, {
                   onReceiveMessage: function (message) {
-                    // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
                     savedReceiveMessage = message;
                   },
                   onReceiveStatus: function (status) {
+                    const requestId: string = JSON.stringify(
```

Comment:
if we decide not to add `request-id` for now then this stuff will need to be removed as well.

Reply:
removed in https://github.com/momentohq/client-sdk-javascript/pull/84/commits/912471012bc256cf275766de7e83ad7e06efd1e9